device/vxlan/unicast.yaml

```diff
@@ -1,45 +1,74 @@
 components:
   schemas:
-    Vxlan.Unicast:
+    Vxlan.V4Tunnel.DestinationIPMode.Unicast:
+      properties:
+        vteps:
+          description: |-
+            List of VTEPs for member VNI(VXLAN Network Identifier)
+          type: array
+          items:
+            $ref: '#/components/schemas/Vxlan.V4Tunnel.DestinationIPMode.Unicast.Vtep'
+
+    Vxlan.V6Tunnel.DestinationIPMode.Unicast:
+      properties:
+        vteps:
+          description: |-
+            List of VTEPs for member VNI(VXLAN Network Identifier)
+          type: array
+          items:
+            $ref: '#/components/schemas/Vxlan.V6Tunnel.DestinationIPMode.Unicast.Vtep'
+
+    Vxlan.Tunnel.DestinationIPMode.Unicast.VtepBase:
       description: >-
-        Configuration of Unicast parameters for VxLAN.
+        DestinationIPMode Unicast VTEP Base
       type: object
       properties:
-        remote_vtep_address:
-          description: >-
-            Remote VXLAN Tunnel End Point address
-          $ref: '#/components/schemas/Remote.Vtep.Address'
-        suppress_arp:
-          description: >-
-            Supress ARP 
-          type: boolean
-          default: false
-        remote_vm_mac:
-          description: >-
-            Remote VM MAC addresses
+        arp_suppression_cache:
+          description: >- 
+            Arp Suppression Cache
           type: array
           items:
-            type: string
-            format: mac
+            $ref : '#/components/schemas/Vxlan.Tunnel.DestinationIPMode.Unicast.ArpSuppressionCache'
+
+    Vxlan.Tunnel.DestinationIPMode.Unicast.ArpSuppressionCache:
+      description: >-
+         ARP suppression reduces ARP broadcast traffic by using ARP Suppression Cache.
+         ARP requests are sent only between the endpoint and the local edge device/VTEP.
+      type: object
+      properties:
+        remote_vm_mac:
+          description: >-
+            Remote VM MAC address binding to Remote VM IPv4 address
+          type: string
+          format: mac
         remote_vm_ipv4:
```

Comment:
We already separated v4_tunnel and v6_tunnel. So we can simply specify ```remote_vm_ip``` as entire IP version have IPv4. 
Please ignore this comment if we have a provision to IPv6 IP within v4_tunnel

Reply:
please ignore this comments as ```remote_vm_ipv4``` will use for v6_tunnel